src/main/java/frc/robot/subsystems/Chassis.java

```diff
@@ -39,7 +39,8 @@
 
     private final double SLOWMODE_COEFFICIENT = 0.5;
     // these need to be set once
-    private final double differpercent = 12 / 25.5; // percent the front needs to move compared to the back, needs to be justed
+    private final double differpercent = 12 / 25.5; // percent the front needs to move compared to the back, needs to be
+                                                    // justed
```

Comment:
nit: maybe put the comment on a separate line to fix the formatting?

Reply:
👍